apps/api-extractor/src/generators/DtsRollupGenerator.ts

```diff
@@ -50,13 +50,15 @@ export class DtsRollupGenerator {
    *
    * @param dtsFilename    - The *.d.ts output filename
    */
-  public static writeTypingsFile(collector: Collector, dtsFilename: string, dtsKind: DtsRollupKind): void {
+  public static writeTypingsFile(
+    collector: Collector, dtsFilename: string, dtsKind: DtsRollupKind, newlineKind: NewlineKind = NewlineKind.CrLf
```

Comment:
Let's make `newlineKind` a required parameter.  If someone forgets to provide the parameter, it's probably a mistake that will quietly introduce a bug.  This is an internal API.